src/javascript/app/pages/user/account/settings/self_exclusion.js

```diff
@@ -268,10 +281,17 @@ const SelfExclusion = (() => {
                 });
             }
 
-            $('#gamstop_info_bottom').setVisibility(is_gamstop_client && date);
+            showWarning(timeout_val || exclude_until_val);
         });
     };
 
+    const showWarning = is_enable => {
```

Comment:
how about `const showWarning = (is_enabled = true) => {`? so that if we call `showWarning()` it will show and we don't need to call `showWarning(true)`, since the function name sounds like it's something that's true by default anyway